docs/compute/_supported_methods_block_storage.rst

```diff
@@ -9,7 +9,8 @@ Provider                              list volumes create volume destroy volume
 `CloudFrames`_                        no           no            no             no            no            no             no             
```

Comment:
Just for the future reference - those documents are updated automatically using `./contrib/generate_provider_feature_matrix_table.py` script.